deepchecks/vision/checks/distribution/train_test_label_drift.py

```diff
@@ -10,23 +10,58 @@
 #
 """Module contains Train Test label Drift check."""
 from copy import copy
-from typing import Dict, Hashable, Callable, Tuple, List, Union
+from typing import Dict, Hashable, Callable, Tuple, List, Union, Any
 
 from plotly.subplots import make_subplots
 
 from deepchecks import CheckResult, ConditionResult
-from deepchecks.vision.base import TrainTestCheck, Context
+from deepchecks.vision.base import Context, TrainTestCheck
 from deepchecks.utils.distribution.plot import drift_score_bar_traces
 from deepchecks.utils.plot import colors
 from deepchecks.vision.dataset import VisionDataset, TaskType
 import numpy as np
 from collections import Counter
 import plotly.graph_objs as go
 
-
 __all__ = ['TrainTestLabelDrift']
 
 
+# TODO: Add label sampling when available
+
+# Functions temporarily here, will be changed when Label and Prediction classes exist:
+def get_bbox_area(label):
+    """Return a list containing the area of bboxes per image in batch."""
+    areas = (label.reshape((-1, 5))[:, 4] * label.reshape((-1, 5))[:, 3]).reshape(-1, 1).tolist()
+    return areas
+
+
+def count_num_bboxes(label):
+    """Return a list containing the number of bboxes per image in batch."""
+    num_bboxes = label.shape[0]
+    return num_bboxes
+
+
+def get_samples_per_class_classification(label):
+    """Return a list containing the class per image in batch."""
+    return label.tolist()
+
+
+def get_samples_per_class_object_detection(label):
+    """Return a list containing the class per image in batch."""
+    return [arr.reshape((-1, 5))[:, 0].tolist() for arr in label]
+
+
+DEFAULT_CLASSIFICATION_LABEL_TRANSFORMERS = [
+    {'name': 'Samples per class', 'method': get_samples_per_class_classification, 'is_continuous': False}
+]
+
+DEFAULT_OBJECT_DETECTION_LABEL_TRANSFORMERS = [
+    {'name': 'Bounding box area distribution', 'method': get_bbox_area, 'is_continuous': True},
+    {'name': 'Samples per class', 'method': get_samples_per_class_object_detection, 'is_continuous': False},
+    {'name': 'Number of bounding boxes per image', 'method': count_num_bboxes, 'is_continuous': False},
+]
+
+
 class TrainTestLabelDrift(TrainTestCheck):
     """
     Calculate label drift between train dataset and test dataset, using statistical measures.
```

Comment:
Please update docstring. Should explain the format and use of alternative_label_transformers

Reply:
Done